_layouts/default.html

```diff
@@ -8,6 +8,7 @@
     <meta name="theme-color" content="#157878">
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <link rel="stylesheet" href="plumedtohtml.css">
```

Comment:
I think that if you put the `plumedtohtml.css` file in the css folder and the js file in a js/script directory
It will be easier to mantain, not for the duplicate files, but for avoiding to forget to put the css/js files in all the directories (that it will be more a nest/tutorial problem that the 5 tests here)

I think the small change should be something like this:
```suggestion
    <link rel="stylesheet" href="/css/plumedtohtml.css">
```
or like this:
```suggestion
    <link rel="stylesheet" href="/assets/css/plumedtohtml.css">
```
 I do not understand where jekyll puts the css and js

Reply:
This was my first thought yesterday.  The css and JavaScript aren't copied by GitHub if you put them in the assets directory.  I thus went for the dumbest option and put them everywhere they are needed.  If you can work out how to get it to work without all the copies, be my guest.

Reply:
Ok, I would start with accepting the pr as its
and then looking for that solution, I think that the problem here is untangible, but could snowball in the other two places

There must be a way, because the css file in the assets is used by jeckyll